docs/Usage.md

```diff
@@ -0,0 +1,119 @@
+## Getting started with the generator 🐣
```

Comment:
Wondering if we want to have _any_ content in the deprecated repository.

It might be best to just have a brand logo, the deprecation message, and a clear link to the new locale.

Reply:
If you look at the new readme, it is almost empty (keep the licence and a link to the archived doc).
The doc is here as an archive for people already using this version but have been moved in a separate file to have a more clear readme.

Reply:
👍